nsv13/code/game/gamemodes/overmap/objectives/rubicon.dm

```diff
@@ -38,8 +39,9 @@
 /datum/overmap_objective/clear_system/rubicon
 	system_name = "Rubicon"
 	extension_supported = TRUE
-	required_players = 4
+	required_players = 10
 
 /datum/overmap_objective/clear_system/dolos
 	system_name = "Dolos Remnants"
-	required_players = 4
+	extension_supported = TRUE //Only if Rubicon is not available
+	required_players = 10
```

Comment:
This system doesn't actually contain any permanent garrison fleets and is fairly likely to end up being a free objective besides for the very minor random ships spawning or a fleet stopping by.

Reply:
Oh you're right, I assumed the remnant was there all the time.